### PR TITLE
make service_account e2e test verify that RootCA cert file exists

### PR DESCRIPTION
**Pull request by mikedanese**

### CODE REVIEW

File: test/e2e/service_accounts.go [hunk 1]
```diff
@@ -37,6 +37,7 @@ var _ = Describe("ServiceAccounts", func() {
 	It("should mount an API token into pods", func() {
 		var tokenName string
 		var tokenContent string
+		var rootCaContent string
```
Comment (roberthbailey) [inline test/e2e/service_accounts.go]: nit: should be named rootCAContent

File: test/e2e/service_accounts.go [hunk 2]
```diff
@@ -78,5 +87,8 @@ var _ = Describe("ServiceAccounts", func() {
 		f.TestContainerOutput("consume service account token", pod, 0, []string{
 			fmt.Sprintf(`content of file "%s/%s": %s`, serviceaccount.DefaultAPITokenMountPath, api.ServiceAccountTokenKey, tokenContent),
 		})
+		f.TestContainerOutput("consume service account token", pod, 1, []string{
```
Comment (roberthbailey) [inline test/e2e/service_accounts.go]: should the text here be consume service account root ca (instead of token)?

**Comment by thockin** [comment]
LGTM - thanks for following through

**Comment by mikedanese** [comment]
Addressed comment's please take another look.

**Comment by thockin** [comment]
LGTM

**Comment by mikedanese** [comment]
@lavalamp @roberthbailey can one of you give a second review?

**Comment by roberthbailey** [comment]
LGTM.

**Comment by roberthbailey** [comment]
Closing and re-opening to re-run shippable.

**Comment by lavalamp** [comment]
lgtm

**Pull request merged**